airflow/hooks/druid_hook.py

```diff
@@ -10,7 +10,7 @@
 from airflow.exceptions import AirflowException
 
 LOAD_CHECK_INTERVAL = 5
-
+DEFAULT_TARGET_PARTITION_SIZE = 5000000
 
```

Comment:
somehow auto-linting reporting is off (I have a ticket open with apache INFRA to switch it back on)

Nit: 2 blank lines before `class` according to PEP8